xml/System.Net.Http/HttpClient.xml

```diff
@@ -140,6 +124,74 @@ If this change is undesirable, you can configure your application to use the old
   
 - By defining an environment variable named `DOTNET_SYSTEM_NET_HTTP_USESOCKETSHTTPHANDLER` and setting it to either `false` or 0.
 
+### Connection Pooling
+
+HttpClient will pool HTTP connections where possible, and use them for more than one request. This can have a significant performance benefit, especially for HTTPS requests, as the connection handshake is only done once.
+
+Connection pool properties can be configured on a <xref:System.Net.Http.HttpClientHandler> or <xref:System.Net.Http.SocketsHttpHandler> passed in during construction, including <xref:System.Net.Http.HttpClientHandler.MaxConnectionsPerServer>, <xref:System.Net.Http.SocketsHttpHandler.PooledConnectionIdleTimeout> and <xref:System.Net.Http.SocketsHttpHandler.PooledConnectionLifetime>.
+
+Todo: Disposing of the HttpClient instance will close the open connections and cancel any pending requests.
+
+### Buffering & Request lifetime
+
+By default, HttpClient methods (except <xref:System.Net.Http.HttpClient.GetStreamAsync>) will buffer the responses from the server, reading all the response body into memory before returning the async result. Those requests will continue until one of the following occurs:
```

Comment:
the xref doesn't render correctly: https://review.docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=netcore-3.1&branch=pr-en-us-3986#buffering--request-lifetime

```suggestion
By default, HttpClient methods (except <xref:System.Net.Http.HttpClient.GetStreamAsync%2A>) will buffer the responses from the server, reading all the response body into memory before returning the async result. Those requests will continue until one of the following occurs:
```